src/index.html

```diff
@@ -42,6 +46,7 @@
     <include src="./partials/products-modal.html"></include>
     <!-- Skrypt jest na samym dole kodu. On juz ma w sobie podłączony SAAS plik. -->
     <script src="./js/index.js"></script>
+    <script src="./js/modal-map.js"></script>
```

Comment:
As you're using open-modal.js, this one is not needed. Can you please remove this file as well? :)